File: frontend/src/model/TrackerAnalyticDto.ts
export interface TrackerAnalyticDto {
    id: string;
    name: string;
    order?: number;
    code: string;
    description?: string;
    trackerAnalyticFields: TrackerAnalyticFieldDto[];
}

export interface TrackerAnalyticFieldDto {
    id: string;
    type: string;
    purpose: string;
    fieldName: string;
}

File: backend/src/Operum.Model/DTOs/Trackers/TrackerAnalyticDto.cs
namespace Operum.Model.DTOs.Trackers
{
    public class TrackerAnalyticDto
    {
        public string Id { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Code { get; set; } = string.Empty;
        public string? Description { get; set; }
        public List<TrackerAnalyticFieldDto> TrackerAnalyticFields { get; set; } = [];
    }
}
